gnomad/utils/transcript_annotation.py

```diff
@@ -283,6 +283,7 @@ def tx_filter_variants_by_csqs(
     ignore_splicing: bool = True,
     filter_to_protein_coding: bool = True,
     vep_root: str = "vep",
+    include_polyphen_prioritization: bool = False,
```

Comment:
is this just to avoid the extra stratification from `process_consequences`?

Reply:
it's because v2 has polyphen on it and v4 doesn't, and process consequences defaults to use polyphen so it fails on the v4 sites HTs